cogs/catgirl/catgirl.py

```diff
@@ -286,6 +302,21 @@ async def add(self, ctx, link: str, description: str = ""):
         else:
             await ctx.send("Added, notified and pending approval. :ok_hand:")
 
+    # [p]nyaa toggle
+    @_nyaa.command(name="toggle")
+    @checks.admin_or_permissions(manage_guild=True)
```

Comment:
let's set this to mods and above.
```suggestion
    @checks.mod_or_permissions(manage_guild=True)
```